skyportal/handlers/api/photometry.py

```diff
@@ -1938,6 +1958,10 @@ def get(self, obj_id):
             include_annotation_info = True
         else:
             include_annotation_info = False
+        if str(include_extinction).lower() in ["true", "t", "1"]:
+            include_extinction = True
+        else:
+            include_extinction = False
```

Comment:
@antoine-le-calloch introduced a helper function to standardize parsing boolean arguments, and updated multiple handlers to use it. Maybe this would be a good opportunity to update the photometry get handler to use that too like the other handlers?